drivers/bluetooth/hci/CMakeLists.txt

```diff
@@ -8,3 +8,4 @@ zephyr_library_sources_ifdef(CONFIG_BT_RPMSG       rpmsg.c)
 zephyr_library_sources_ifdef(CONFIG_BT_SPI         spi.c)
 zephyr_library_sources_ifdef(CONFIG_BT_STM32_IPM   ipm_stm32wb.c)
 zephyr_library_sources_ifdef(CONFIG_BT_USERCHAN    userchan.c)
+zephyr_library_sources_ifdef(CONFIG_BT_CYW43XXX    cyw43xxx.c)
```

Comment:
Would be good to preserve alphabetical order.

Reply:
done